webcompat/static/js/lib/issue-wizard-popup.js

```diff
@@ -3,39 +3,58 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import $ from "jquery";
+import Mousetrap from "Mousetrap";
 import { sendAnalyticsEvent } from "./wizard/analytics.js";
 
 function Popup() {
   this.init = function () {
+    this.closeButtons = document.querySelectorAll(".popup-modal__close");
+    this.modalTriggers = document.querySelectorAll(".popup-trigger");
+    this.overlay = document.querySelector(".popup-overlay");
+
     this.setUpEvents();
   };
 
   this.setUpEvents = function () {
-    this.popupHandler();
+    this.modalTriggers.forEach(
+      function (trigger) {
+        trigger.addEventListener("click", this.openModal.bind(this));
+      }.bind(this)
+    );
+
+    this.closeButtons.forEach(
+      function (btn) {
+        btn.addEventListener("click", this.closeModal.bind(this));
+      }.bind(this)
+    );
+
+    Mousetrap.bind("esc", this.closeModal.bind(this));
+  };
+
+  this.openModal = function (e) {
+    e.preventDefault();
+
+    var popupTrigger = e.target.dataset.popupTrigger;
+    var popupModal = document.querySelector(
+      '[data-popup-modal="'.concat(popupTrigger, '"]')
+    );
+    sendAnalyticsEvent(
+      // transform trigger into camelCase and send as our event
+      popupTrigger.replace(/-([a-z])/g, function (match) {
+        return match[1].toUpperCase();
+      }),
```

Comment:
Since analytics goal tracking for this step is set up as `whatIsCompat`, we probably need to keep it that way. It's possible to change the label, but then previous weeks data will still have the old label and won't be reflected in the statistics :(

We can probably change `data-popup-trigger="what-is-webcompat"` to `data-popup-trigger="what-is-compat"` (and same with `data-popup-modal`)

Reply:
oh yeah! i didn't even notice the difference there, thanks :)